src/MLJXGBoostInterface.jl

```diff
@@ -775,22 +280,29 @@ MMI.load_path(::Type{<:XGBoostRegressor}) = "$PKG.XGBoostRegressor"
 MMI.input_scitype(::Type{<:XGBoostRegressor}) = Table(Continuous)
 MMI.target_scitype(::Type{<:XGBoostRegressor}) = AbstractVector{Continuous}
 MMI.docstring(::Type{<:XGBoostRegressor}) =
-    "The XGBoost gradient boosting method, for use with "*
-    "`Continuous` univariate targets. "
+"The XGBoost gradient boosting method, for use with "*
+"`Continuous` univariate targets. "
 
 MMI.load_path(::Type{<:XGBoostCount}) = "$PKG.XGBoostCount"
 MMI.input_scitype(::Type{<:XGBoostCount}) = Table(Continuous)
 MMI.target_scitype(::Type{<:XGBoostCount}) = AbstractVector{Count}
 MMI.docstring(::Type{<:XGBoostCount}) =
-    "The XGBoost gradient boosting method, for use with "*
-    "`Count` univariate targets, using a Poisson objective function. "
+"The XGBoost gradient boosting method, for use with "*
```

Comment:
To improve readability of the auto-generated header for the docstrings, please add the following trait declaration:

```
MMI.human_name(::Type{<:XGBoostRegressor}) = "eXtreme gradient boosting regressor"
```
or similar. And ditto the other models.